internal/module/openapi.go

```diff
@@ -66,32 +67,40 @@ func helmFormatModuleImages(m *Module, rawValues map[string]any) (chartutil.Valu
 	return top, nil
 }
 
-func GetModulesImagesDigests(modulePath string) (modulesDigests map[string]any, err error) {
+func GetModulesImagesDigests(moduleName, modulePath string) (modulesDigests map[string]any, err error) {
 	var (
 		search bool
```

Comment:
As far as I can see, is it possible to do without this variable?

Reply:
Image digests file has simpler structure in external modules for deckhouse. It contains structure like:
```
{
  "imageName1": "sha256...",
  "imageName2": "sha256..."
  ...
}
```

Reply:
So basically we need to pass module name in lower camel case in order to add it to default digests map